libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/internal/ui/RouteLineComponentTest.kt

```diff
@@ -208,7 +208,14 @@ class RouteLineComponentTest {
 
         routesObserverSlot.captured.onRoutesChanged(routesUpdateResult)
 
-        verify { mockApi.setNavigationRouteLines(any(), any(), capture(callbackSlot)) }
+        verify {
+            mockApi.setNavigationRouteLines(
+                any(),
+                0,
+                any(),
+                capture(callbackSlot)
+            )
+        }
```

Comment:
```suggestion
        verify {
            mockApi.setNavigationRouteLines(
                any(),
                any<List<AlternativeRouteMetadata>>(),
                capture(callbackSlot)
            )
        }
```
Why not keep the changes from the original commit? Same everywhere below.

Reply:
Let's leave as is to avoid CI runs.